public/editor/scripts/main.js

```diff
@@ -52,7 +52,7 @@ require(["jquery", "bowser"], function($, bowser) {
 
   Bramble.once("error", onError);
 
-  function init(BrambleEditor, Project, SSOOverride, ProjectRenameUtility) {
+  function init(BrambleEditor, Project, SSOOverride) {
```

Comment:
now that you've removed `ProjectRenameUtility` here, you should also remove `fc/project-rename` from the array on line 79 since we don't use it anymore.